pwnlib/util/misc.py

```diff
@@ -248,17 +254,23 @@ def run_in_new_terminal(command, terminal = None, args = None):
 
     log.debug("Launching a new terminal: %r" % argv)
 
-    pid = os.fork()
-
-    if pid == 0:
-        # Closing the file descriptors makes everything fail under tmux on OSX.
-        if platform.system() != 'Darwin':
```

Comment:
This was added in 60e5a8079074eb3fe967148ce33144853bb01a13. I wasn't able to repro on my Mac OS X machine with tmux 2.8, but should be straightforward to add back if it is still an issue.